src/web/routes/application/children-dob/predicates.test.js

```diff
@@ -0,0 +1,22 @@
+const test = require('tape')
+const { keyDoesNotContainIndex, isNotChildEntry, isChildEntry } = require('./predicates')
+
+test('keyDoesNotContainIndex()', (t) => {
+  t.equal(keyDoesNotContainIndex(1)('value', 'childDob-1-day'), false)
```

Comment:
can we test: `t.equal(keyDoesNotContainIndex(1)('value', 'childDob-11-day'), false)`?